packages/cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal.ts

```diff
@@ -2339,6 +2395,78 @@ export default class CBioPortalAPIInternal {
                 return response.body;
             });
         };
+    fetchClinicalEventsMetaUsingPOSTURL(parameters: {
```

Comment:
@haynescd can you help review the changes to the api here.  did we introduce this new endpoint?